backend/routes/appointments/viewAppointmentsRouter.js

```diff
@@ -53,6 +111,7 @@ module.exports = ({ prisonApi, whereaboutsApi, logError }) => async (req, res) =
 
   const appointmentsEnhanced = appointments
     .filter(appointment => (type ? appointment.appointmentTypeCode === type : true))
+    .filter(appointment => !VLPreAppIds.includes(appointment.id) && !VLPostAppIds.includes(appointment.id))
```

Comment:
I think this can be achieved by just looking at the VLB booking type, then we perhaps don't need to store the id's